german/metamath.tex

```diff
@@ -2554,7 +2555,7 @@ \section{Ihr erster Beweis}\label{frstprf}
 
 Ein oft nützlicher Befehl ist \texttt{minimize{\char`\_}with*/brief}, der versucht, den Beweis zu verkürzen.  Er kann den Prozess des Beweisens effizienter machen, indem er Sie einen etwas "`schlampigen"' Beweis schreiben lässt und ihn dann durch einige feine Optimierungsdetails für Sie bereinigt (obwohl er keine Wunder vollbringen kann, wie z.B. die Umstrukturierung des gesamten Beweises).
 
-\section{\sloppy Hinweise zur Bearbeitung einer Daten\-basis}
+\section{Hinweise zur Bearbeitung einer Daten\-basis}
```

Comment:
Without the \sloppy command, LaTex does not wrap "Datenbasis", but creates a single line which is a little bit longer than the block size. I would suggest to insert \texorpdfstring{\\}{} before  "Datenbasis" (wraps the line before  "Datenbasis",).